CHANGELOG.md

```diff
@@ -1,5 +1,96 @@
 # Changelog
 
+## [0.23.0] - 2024-08-26
+
+### Breaking
+
+- Introduce tree-sitter-language crate for grammar crates to depend on (<https://github.com/tree-sitter/tree-sitter/pull/3069>)
+- Revert interning of a sequence or choice of a single rule (<https://github.com/tree-sitter/tree-sitter/pull/3548>)
```

Comment:
Technically, that shouldn't be here since it reverts a commit _after_ the previous release. ;)

Reply:
:see_no_evil: true